plugins/scaffolder-actions/scaffolder-backend-module-http-request/src/actions/run/backstageRequest.ts

```diff
@@ -103,11 +110,14 @@ export function createHttpBackstageAction(options: {
       const { input } = ctx;
       const token = ctx.secrets?.backstageToken;
       const { method, params } = input;
+      const logging = input.logging ?? true;
       const url = await generateBackstageUrl(discovery, input.path);
 
-      ctx.logger.info(
```

Comment:
Maybe it would be worthwhile to leave some logging lines still in place, just omitting the sensitive information from being logged.

Reply:
Something like this maybe?
```ts
if (logRequestPath) {
  ctx.logger.info(
    `Creating ${method} request with ${this.id} scaffolder action against ${input.path}`,
  );
} else {
  ctx.logger.info(
    `Creating ${method} request with ${this.id} scaffolder action`,
  );
} 
```

Reply:
yup, looks good